src/utils.ts

```diff
@@ -51,3 +51,8 @@ export function getEventId(event: ethereum.Event): string {
     event.transaction.hash.toHex() + "_" + event.transactionLogIndex.toString()
   );
 }
+
+export function nRLCToRLC(value: BigInt): BigInt {
+  let divisor = BigInt.fromI32(1000000000);
+  return value.div(divisor);
+}
```

Comment:
Maybe you you can re-use this function name for consistency 
https://github.com/iExecBlockchainComputing/PoCo-subgraph/blob/619bfa67720509cb0f3c9d7b3eeff9496681c8ed/src/utils.ts#L175